src/Apps/Sell/Routes/DimensionsRoute.tsx

```diff
@@ -25,10 +28,10 @@ const FRAGMENT = graphql`
 `
 
 const Schema = Yup.object().shape({
-  width: Yup.string().trim(),
-  height: Yup.string().trim(),
+  width: Yup.string().required().trim(),
+  height: Yup.string().required().trim(),
```

Comment:
Have we already made a decision on whether to make the dimensions required or not? Just found this [comment](https://www.figma.com/design/E6rVm1tiSI4PW3EzibKKZw?node-id=2036-30666#839058696) in Figma:

Reply:
Ah, no, i don't think we have
Sorry, didn't see the comment, will remove the` required()`